app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<section class="min-h-screen bg-gray-200 flex flex-col items-center w-full">
+  <div class="bg-main w-full text-white pt-2">
+    <%= link_to category_transactions_path(@category), class: "text-white ps-3" do %>
+      <i class="fa-solid fa-arrow-left fa-lg"></i>
+    <% end %>
+    <div class="h-10 bg-main w-full text-white flex justify-center items-end pb-2">
+      <h1 class="font-bold">Add a New Transaction</h1>
+    </div>
+  </div>
+  <div class="mt-4 px-3 w-full gap-2">
+        <%= form_with(model: [@category, @transaction], url: category_transactions_path(@category)) do |f| %>
+          <div class="flex flex-col">
+            <%= f.label :name %>
+            <%= f.text_field :name, autofocus: true, class: "w-full px-2 py-2" %>
+            
+            <%= f.label :amount %>
+            <%= f.number_field :amount, class: "p-2" %>
+          </div>
+          <div class="mt-3">
+            <%= @category.name %>
+            <%= f.label :category %>
```

Comment:
- [ Required ] I think it would be better if you include a `select` box, not a `label` that allows a user to select a category that the transaction should fall in as requested by the previous reviewer. Currently, your application pre-selects the category for the user at times a user might want to select a different category. 


## Project requirements
![Screenshot from 2023-07-15 21-42-09](https://github.com/SamTush/budget/assets/16392046/ce094cc4-3f4c-43ef-8d87-f683fd106da8)

## Your design
![Screenshot from 2023-07-15 21-32-01](https://github.com/SamTush/budget/assets/16392046/b626baf4-4ea8-4f6d-9af3-769e7896b3be)